docs/design-docs/overview.md

```diff
@@ -112,11 +112,11 @@ Here're the related concepts, nccl benchmark is used as an example.
 
 2. Mode
 
-    One module may have several modes when running, each mode has corresponding method to run, e.g., local mode is running inside one node, mpi mode is running on all nodes but the command is only executed on one node, pair mode is running between two nodes for every combination, etc. The nccl module may have local mode and mpi mode.
+    One module may have several modes when running, each mode has corresponding method to run, e.g., local mode is running inside one node, mpi mode is running on all nodes but the command is only executed on one node, mpi-parallels mode is running nodes for between specified nodes group according to different pattern combination in parallel, etc. The nccl module may have local mode, mpi mode and mpi-parallels mode.
```

Comment:
@abuccts do you list all modes somewhere?

Reply:
If not, we may rewrite this paragraph and include all mode

Reply:
I have found a [doc](https://github.com/microsoft/superbenchmark/blob/main/docs/superbench-config.mdx) including all nodes. will also update here.

Reply:
> mpi-parallels mode is running nodes for between specified nodes group according to different pattern combination in parallel

If I understand correctly, mpi-parallels mode is running tests with the nodes in specified nodes group in parallel. The nodes group is constructed according to different pattern combination.
If so, can we also rephrase this? It's a bit confusing.